CMakeLists.txt

```diff
@@ -28,7 +33,7 @@ include_directories(
 
 list(APPEND LIBS
     ${IBP_LIBRARY} ${GOP_LIBRARY} ${CZMQ_LIBRARY} ${TOOLBOX_LIBRARY}
-    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES}
+    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES} gssapi_krb5
```

Comment:
It depends on how 0mq gets built. -Alan

On 10/29/2015 3:17 PM, Andrew Melo wrote:
>
> In CMakeLists.txt 
> <https://github.com/accre/lstore-gridftp/pull/3#discussion_r43438469>:
>
> > @@ -28,7 +33,7 @@ include_directories(
> >
> >  list(APPEND LIBS
> >      ${IBP_LIBRARY} ${GOP_LIBRARY} ${CZMQ_LIBRARY} ${TOOLBOX_LIBRARY}
> > -    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES}
> > +    ${JERASURE_LIBRARY} ${ZMQ_LIBRARY} ${FUSE_LIBRARIES} gssapi_krb5
>
> Ah okay. I don't think we need it at all. Let me drop it and see how 
> it goes.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/accre/lstore-gridftp/pull/3/files#r43438469>.
>